Source/CesiumRuntime/Private/CesiumEncodedMetadataComponent.cpp

```diff
@@ -590,22 +569,23 @@ void UCesiumEncodedMetadataComponent::GenerateMaterial() {
       UMaterialExpressionTextureObjectParameter* PropertyArray =
           NewObject<UMaterialExpressionTextureObjectParameter>(
               this->TargetMaterialLayer);
+
+      FString propertyName = createHlslSafeName(property.Name);
+
       PropertyArray->ParameterName =
-          FName("FTB_" + featureTable.Name + "_" + property.Name);
+          FName("FTB_" + featureTable.Name + "_" + propertyName);
```

Comment:
Here only the property name is sanitized, instead of the whole parameter name "FTB_featureTableName_propertyName". I'm not sure material parameter names need to be sanitized in general, but either way this is inconsistent with how the names are sanitized in `CesiumGltfComponent::SetMetadataFeatureTableParameterValues`, there the entire parameter name including the feature table name is sanitized.